src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy

```diff
@@ -103,7 +103,7 @@ class PalantirDockerPlugin implements Plugin<Project> {
                         group = 'Docker'
                         description = "Tags Docker image with tag '${tagName}'"
                         workingDir dockerDir
-                        commandLine 'docker', 'tag', '--force=true', ext.name, computeName(ext.name, tagName)
+                        commandLine 'docker', 'tag', ext.name, computeName(ext.name, tagName)
```

Comment:
Docker Engine users pulling base images from Docker Hub ought to be on =>1.10 now, to avoid image ID incompatibilities.  Would you re-consider merging this PR?

I'm building my images with Atlassian Bamboo.  `gradle dockerTag` returns with a 0 exit code, but the warning message from `docker` outputs to stderr.  It seems that Bamboo is picking up the output on stderr and fails the build, even though the `gradle` command succeeds.

I created an `Exec` task in my build based on the code in `PalantirDockerPlugin` without the `--force=true` option, and my Bamboo build now succeeds.

**Environment**
- **OS:** CentOS 7
- **Docker Engine:** 1.10.2
- **Gradle**: 2.10 wrapper
- **Bamboo:** 5.9.9